Source/VMware.vSphereDSC/DSCResources/VMHostService.ps1

```diff
@@ -0,0 +1,155 @@
+
+Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#>
+
+[DscResource()]
+class VMHostService : VMHostBaseDSC {
+    <#
+    .DESCRIPTION
+
+    Host Service key.
+    #>
+    [DscProperty(Mandatory)]
+    [string]$Key
+
+    <#
+    .DESCRIPTION
+
+    Host Service Policy.
+    #>
+    [DscProperty()]
+    [ServicePolicy]$Policy
+
+    <#
+    .DESCRIPTION
+
+    Host Service Running State.
+    #>
+    [DscProperty()]
+    [bool]$Running
+
+    <#
+    .DESCRIPTION
+
+    Host Service Label.
+    #>
+    [DscProperty(NotConfigurable)]
+    [string]$Label
+
+    <#
+    .DESCRIPTION
+
+    Host Service Required flag.
+    #>
+    [DscProperty(NotConfigurable)]
+    [bool]$Required
+
+    <#
+    .DESCRIPTION
+
+    Firewall rule set.
+    #>
+    [DscProperty(NotConfigurable)]
+    [string[]]$Ruleset
+
+    [void]Set() {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+
+        $this.UpdateVMHostService($vmHost)
+    }
+
+    [bool]Test() {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+
+        return !$this.ShouldUpdateVMHostService($vmHost)
+    }
+
+    [VMHostService]Get() {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $vmHostService = [VMHostService]::new()
+        $vmHostService.Server = $this.Server
+
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+        $this.PopulateResult($vmHost, $vmHostService)
+
+        return $vmHostService
+    }
+
+    <#
+    .DESCRIPTION
+
+    Returns a boolean value if the VMHostService needs to be updated.
```

Comment:
Change the comment to "Returns a boolean value indicating if the VMHostService should be updated.".